app/models/manageiq/providers/azure/cloud_manager/provision/cloning.rb

```diff
@@ -183,5 +183,9 @@ def start_clone(clone_options)
 
       File.join(azure.subscription_id, vm.resource_group.downcase, vm.type.downcase, vm.name)
     end
+  rescue StandardError => e
```

Comment:
Can we rescue a more specific exception class?  Was there an AzureArmrest exception that was raised in this case?

Reply:
Yeah, it was an Azure::Armrest::BadRequestException, so I can change it to 
`rescue Azure::Armrest::BadRequestException => e`